### ISSUE TITLE
Questions about training parameters

### ISSUE THREAD

**Issue by ShengnanDing** [pouya-bashivan/EEGLearn#51]
Hello professor!
Now I'm running this code using pytorch.It confuses me that if I want to reach your performance what batch size I should set.Because useing VDelv's code and cross validation 13 can not reach your performance.In your paper you said that batch size is 20,but This code I see that is 32.
I use parameters: batch size = 32, num_epoch = 10 and optimizer is Adam, learning rate = 0.001 ,and result is 
0.6 0.825 0.88 0.98 0.99 1 0.97 0.99 0.99 0.97 0.27 0.81 0.67
and your parper results is 
0.889 0.765 0.933 0.99 1 0.98 1 0.985 0.99 0.968 0.965 0.91 0.468

Hope to hear from you soon.
Thank you very much.

**Comment by pbashivan** [comment]
There are sometimes differences from one run to another but in your case it seems like for two subjects the results are much lower. 
In general changing the batch size to 32 should not be changing anything much. Are you matching the number of epochs/iterations so on? Are you using the same architecture? Transforming the dataset the same way? etc
